packages/terra-consumer-nav/src/components/nav-items/NavItem.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './NavItem.scss';
+import NavToggler from '../nav-toggler/NavToggler';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  // do we need these...
+  slug: PropTypes.string,
+  navType: PropTypes.oneOf(['GROUPING', 'EXTERNAL', 'APPLICATION', 'MODAL']),
```

Comment:
can we keep these lowercase? You can map these to uppercase constant variables if you wish. It's just more consistent with terra